### DISCUSSION TITLE
Forge Server on Docker on Synology NAS will not set FORGE_VERSION

### DISCUSSION THREAD

**Discussion by Sgt-Shadow3600** [itzg/docker-minecraft-server#2256] [Q&A]
I'm trying to set forge to a specific version and it will not change from the default one of 43.2.0 

I have there setting in Advanced Settings now

VARIABLE:                    VALUE:
TYPE                             FORGE
VERSION                      1.19.2
FORGE_VERSION          43.2.3

When I look at the instruction is shows to put it on the same line but on the Synology docker you can not do that. As its all boxes you enter.

**Comment by itzg** [comment]
That should work correctly with those variables. Make sure you have pulled the latest image. If it still isn't working as expected, set `DEBUG` to "TRUE" and provide the container startup logs.

**Reply by Sgt-Shadow3600** [reply]
So I copied the settings and then enabled Debugging. and it actual started working on the test so i will just rename that one and then delete the one that wasn't working 

Thanks
